apps/server/src/adapters/OscAdapter.ts

```diff
@@ -15,13 +15,13 @@ export class OscServer implements IAdapter {
     this.osc.on('error', console.error);
 
     this.osc.on('message', (msg) => {
-      // message should look like /ontime/{path} {args} where
+      // message should look like /ontime/{path}/{params?} {args} where
       // ontime: fixed message for app
       // path: command to be called
       // args: extra data, only used on some API entries (delay, goto)
 
       // split message
-      const [, address, path] = msg[0].split('/');
+      const [, address, path, ...params] = msg[0].split('/');
       const args = msg[1];
```

Comment:
if there is no args this throws `Error: uncaught exception TypeError: Cannot read properties of undefined (reading 'elements')`

Reply:
I thought `args` should ever exist? I was not even able to send a OSC message without a payload